src/admin/components/views/collections/Edit/Default.tsx

```diff
@@ -78,14 +81,29 @@ const DefaultEditView: React.FC<Props> = (props) => {
     isEditing && `${baseClass}--is-editing`,
   ].filter(Boolean).join(' ');
 
-  const onSave = useCallback((json) => {
+  const onSave = useCallback(async (json) => {
     if (typeof onSaveFromProps === 'function') {
+      if (auth && id === user.id) {
+        const request = await requests.post(`${serverURL}${api}/${collection.slug}/refresh-token`, {
```

Comment:
I expected `auth.refreshCookie()` here and and updated `refreshCookie` method that always executes the refresh.